features/machine/machine.feature

```diff
@@ -128,21 +128,12 @@ Feature: Machine features testing
 
     Given I clone a machineset and name it "machineset-clone-27609"
 
-    Then I run the :get admin command with:
-     | resource      | machineset             |
-     | resource_name | machineset-clone-27609 |
-     | namespace     | openshift-machine-api  |
-     | o             | yaml                   |
-    And I save the output to file> new_machineset.yaml
-
-    And I replace lines in "new_machineset.yaml":
-     | publicIp: null| publicIp: true |
-    Then the step should succeed
-
-    When I run the :replace admin command with:
-     | f | new_machineset.yaml |
-    And the output should match:
-     | [Rr]eplaced |
+    Then I run the :patch admin command with:
+     | resource      | machineset                                         |
+     | resource_name | machineset-clone-27609                             |
+     | type          | merge                                              |
+     | p             | {"metadata":{"providerSpec": {"publicIP": "true"}}}|
+    And the step should succeed
```

Comment:
Given /^(?:(as admin) )?I successfully#{OPT_WORD} patch resource "(.*)\/(.*)" with:$/ do |as_admin, patch_type, resource_type, resource_name, table|

Reply:
Thanks @liangxia  for review , made changes as suggested and validated them , looking good - https://mastern-jenkins-csb-openshift-qe.cloud.paas.psi.redhat.com/job/Runner-v3/93838/console 

Please help merge this